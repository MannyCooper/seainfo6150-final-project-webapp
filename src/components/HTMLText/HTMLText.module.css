@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');

.content{
    margin: 40px auto;
    width: 773px;
    word-wrap: break-word;
}

.content img{
    width: 100%;
    border-radius: 15px;
    margin: 20px 0;
}

.content p{
    font-family: 'Lato', sans-serif;
    line-height: 1.6;
    margin-bottom: 15px !important;
    font-size: 16px;
}

.content a{
    color: #ff375f
}

@media only screen and (max-width: 1380px){
    .content{
        max-width: 653px;
    }
}

@media only screen and (max-width: 1200px){
    .content{
        width: 560px;
    }
}

@media only screen and (max-width: 768px){
    .content {
        max-width: 340px;
        width: 87.5%;
    }
    .content p{
        font-size: 15px;
    }
}
