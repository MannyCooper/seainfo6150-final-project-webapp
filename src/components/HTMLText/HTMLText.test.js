 /* global render */
 import React from "react";
 import HTMLText from "./HTMLText.jsx";
  
 const testText = "<p><a href='https://www.pixijs.com/'>PixiJS</a></p>";
 it("renders the HTMLText component correctly", () => {
   const { container } = render(<HTMLText text={testText} />);
   expect(container).toMatchSnapshot();
 });
 