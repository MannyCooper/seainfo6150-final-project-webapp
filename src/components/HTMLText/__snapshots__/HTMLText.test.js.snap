// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the HTMLText component correctly 1`] = `
<div>
  <div
    class="content"
  >
    <p>
      <a
        href="https://www.pixijs.com/"
      >
        PixiJS
      </a>
    </p>
  </div>
</div>
`;
